### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by anasbassi**
In this project the following requirements were implemented;
- Create a app.rb file to present functionalities.
- Created main file to execute the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    puts 'Select a book from the following list by number'
+    list_books
+    book_index = gets.chomp.to_i
+    puts 'Select a person from the following list by number (not id)'
+    list_people
+    person_index = gets.chomp.to_i
+    print "Date: #{Date.today}"
+    date = gets.chomp
+    rental = Rental.new(date, @books[book_index], @people[person_index])
+    @rentals.push(rental)
```
Comment (uche-inyama) [inline app.rb]: On line 48, kindly inspect the way you arranged your arguments, with how you defined the parameters in the rental.rb file.

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+  def create_rental
+    puts 'Select a book from the following list by number'
+    list_books
+    book_index = gets.chomp.to_i
+    puts 'Select a person from the following list by number (not id)'
+    list_people
+    person_index = gets.chomp.to_i
+    print "Date: #{Date.today}"
+    date = gets.chomp
```
Comment (uche-inyama) [inline app.rb:48]: Since you are using the date module, kindly require it in this file.

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    end
+  end
+
+  def list_rentals
+    puts 'Enter ID of the person'
+    list_people
+    person_id = gets.chomp.to_i
+    person = @people.select { |p| p.id == person_id }[0]
+    person.rentals.each_with_index { |rental, i| puts "#{i}) Book: #{rental.book.title}, Date: #{rental.date}" }
+  end
```
Comment (uche-inyama) [inline app.rb:74]: Also, kindly redefine line 72, look closely and correct the error. Compare it with person class.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
### Hi @anasbassi 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

After running your application, I encountered errors when I wanted to create and list rental. 

#### Kindly go through the comments I left blow. :point_down: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
## Hi @anasbassi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**